distribution/lib/Standard/Base/0.0.0-dev/src/Any.enso

```diff
@@ -20,6 +20,49 @@ from project.Function import const
    be used in that position.
 @Builtin_Type
 type Any
+    ## GROUP Conversions
+       Generic conversion of an arbitrary Enso value to requested type
+
+       Arguments:
```

Comment:
It would probably be good to mention that `to` exists iff `from` is present. I don't think this is obvious from the current description.

Reply:
Done in 32301b4